packages/lab/src/navigation-item/NavigationItem.tsx

```diff
@@ -89,13 +91,20 @@ export const NavigationItem = forwardRef<HTMLDivElement, NavigationItemProps>(
       onExpand?.(event);
     };
 
+    const buttonRef = useRef<HTMLButtonElement | null>(null);
+
+    useEffect(() => {
+      expanded && buttonRef.current?.focus();
```

Comment:
I don't think this is the solution to focus being lost.

It's very likely due to `ConditionalWrapper` where a new Button is created. Can try refactor the code so `Button` stays the same

Reply:
Should be fixed now 😄